drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -777,7 +777,6 @@ void RasterizerSceneGLES3::_draw_sky(RID p_env, const Projection &p_projection,
 	ERR_FAIL_COND(p_env.is_null());
 
 	Sky *sky = sky_owner.get_or_null(environment_get_sky(p_env));
-	ERR_FAIL_NULL(sky);
```

Comment:
When doing a custom sky pass for fog we can continue without the sky. This is used when you have fog, but are using a custom color background.